test/project/config/crds/kustomizeconfig.yaml

```diff
@@ -0,0 +1,14 @@
+# This file is static and is generated by kubebuilder
```

Comment:
can we add a comment explaining why this exists (e.g. "this is needed to teach kustomize about how to set variables in CRDs...")